docs/blog/2019-08-14-strivectin-case-study/index.md

```diff
@@ -0,0 +1,134 @@
+
+After defining and populating our CMS with sample data, we were onto the Gatsby phase. Keep in mind that we didn't have any templates like a typical Shopify theme. You would think that it would be slower to start from scratch, but it had the opposite effect. Implementing with Gatsby enabled us to developer faster. We weren't bound by templates like a typical Shopify theme. We were able to reuse queries and components across pages.
+
+After completing the majority of the implementation, we moved onto integration with Shopify. The Storefront API made it easy to create checkouts. We use the online store for account management, so we had to theme it. We mirrored the Gatsby site for the account management pages.
+
+StriVectin also offers subscription products that required a split checkout. Recharge, a third party, handles these recurring transactions. We used Firebase HTTP functions to create checkouts against their API.
+
+We used other Firebase functions for triggering builds (webhooks), testing lighthouse scores, and notifications. Our build triggers come from webhooks, pull requests, and builds.
+
+## Launch
+
+Launch was effortless. We had the site living on a subdomain of strivectin.com so we were able to conduct thorough testing of cross-domain tracking, cookies, and checkout processes with Shopify and ReCharge.
+
+Felicity Sissener, the Chief Digital Officer at StriVectin had this to say about launch:
+
+<Pullquote>
+“This has also been the smoothest cut-over of my entire career- your team and work are top notch.”
+</Pullquote>
+
+We confidently ship code to production many times per day. At the time of writing this, we have not had a single site outage.
+
+StriVectin’s hosting costs have gone from \$2,000/month to just a few dollars per day. The servers will be decommissioned very soon.
+
+<figure>
+  <video autoplay muted loop width="300">
+    <source src="https://media.giphy.com/media/DC4g3SGNJpC/giphy.mp4"/>
+  </video>
+  <figcaption>Server smashing time</figcaption>
+</figure>
+
+Feature development and maintenance is much simpler. The codebase was around 20,000 files on Magento and went down to around 300.
+
+## Final Thoughts
+
+We are still less than 30 days out from launch so we have a lot of time for standard eCommerce metrics such as conversion rate and revenue per visitor to stabilize but the results so far look great. One KPI that is not discussed much in headless/PWA discussions is the impact on organic SEO traffic. We have seen an immediate and sustained increase of 20% in organic traffic to the site.
```

Comment:
*we have had a lot of time for standard

Reply:
I think this reads a little confusing. What we are saying is that less than 30 days isn't enough time for accurately measuring changes in eCommerce metrics.

Alternatively, we could say "We are still less than 30 days out from launch which means the typical eCommerce KPIs like conversion rate and revenue per visitor have not stabilized due to traffic and marketing changes during the cutover."

Reply:
> We are still less than 30 days out from launch which means the typical eCommerce KPIs like conversion rate and revenue per visitor have not stabilized due to traffic and marketing changes during the cutover.

This reads much better

Reply:
Thanks. Updated in 046c737